evalData/testdata_insertion.py

```diff
@@ -1,23 +1,25 @@
-import http.client
 import io
 import os
 import json
+import requests
 
 if __name__ == "__main__":
-    conn = http.client.HTTPConnection("localhost", 5000)
+    session = requests.Session()
+    data = {"email": "admin@knex.com", "password": "admin"}
+    session.post("http://localhost:3000/api/users/login", data=data)
```

Comment:
Oh, missed that part (BE no longer being exported on port 5000), perfectly makes sense then.